demo/jenkins.yaml

```diff
@@ -0,0 +1,43 @@
+jenkins:
+  systemMessage: "Jenkins configured automatically by Jenkins Configuration as Code Plugin\n\n"
```

Comment:
would be nice to reference your demo and to link to the plugin readme

Reply:
@oleg-nenashev how to add hyperlink to display in jenkins?

Reply:
@oleg-nenashev How to add a hyperlink to display as a String in Java?

Reply:
Just add it as a text for now. There are different text rendering engines in Jenkins ("MarkupFormatter"s), and I would recommend to avoid hyperlinks unless you explicitly specify a formatter in the config YAML. E.g. https://plugins.jenkins.io/antisamy-markup-formatter